token-lending/README.md

```diff
@@ -71,51 +71,52 @@ This is optional! You can skip these steps and use the [Token Lending CLI](./cli
    ```
    This pubkey will be your Program ID.
 
-1. Open `./token-lending/program/src/lib.rs` in your editor. In the line
+7. Open `./token-lending/program/src/lib.rs` in your editor. In the line
    ```rust
    solana_program::declare_id!("6TvznH3B2e3p2mbhufNBpgSrLx6UkgvxtVQvopEZ2kuH");
    ```
    replace the Program ID with yours.
 
-1. Build the program binaries:
+8. Build the program binaries:
    ```shell
    cargo build
    cargo build-bpf
    ```
 
-1. Prepare to deploy to devnet:
+9. Prepare to deploy to devnet:
    ```shell
    solana config set --url https://api.devnet.solana.com
    ```
 
-1. Score yourself some sweet SOL:
-   ```shell
-   solana airdrop -k owner.json 10
-   solana airdrop -k owner.json 10
-   solana airdrop -k owner.json 10
-   ```
-   You'll use this for transaction fees, rent for your program accounts, and initial reserve liquidity.
-
-1. Deploy the program:
-   ```shell
-   solana program deploy \
-     -k owner.json \
-     --program-id lending.json \
-     target/deploy/spl_token_lending.so
-
-   # Program Id: 6TvznH3B2e3p2mbhufNBpgSrLx6UkgvxtVQvopEZ2kuH
-   ```
-   If the deployment doesn't succeed, follow [this guide](https://docs.solana.com/cli/deploy-a-program#resuming-a-failed-deploy) to resume it.
-
-1. Wrap some of your SOL as an SPL Token:
-   ```shell
-   spl-token wrap \
-      --fee-payer owner.json \
-      10.0 \
-      -- owner.json
-
-   # Wrapping 10 SOL into AJ2sgpgj6ZeQazPPiDyTYqN9vbj58QMaZQykB9Sr6XY
-   ```
-   You'll use this for initial reserve liquidity. Note the SPL Token account pubkey (e.g. `AJ2sgpgj6ZeQazPPiDyTYqN9vbj58QMaZQykB9Sr6XY`).
-
-1. Use the [Token Lending CLI](./cli/README.md) to create a lending market and add reserves to it.
\ No newline at end of file
+10. Score yourself some sweet SOL:
+    ```shell
```

Comment:
please revert the superfluous indent changes from here to end

Reply:
Fixed!